portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,58 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Diana's Portfolio</h1>
+      <p>I am STEP intern studying <b>Computer Science</b> in
+         the University of Texas at El Paso, currently 
+         working remotely from Mexico. Here's a picture
+         of one of my dogs, he's name is Remmy! </p><br>
+      <img src = "/myImages/remmy.jpeg" alt = "Remmy" width = "200" height = "300" /> <br>
+
+      <h2>My Background</h2>
+      <p> I have the oportunity to study abroad bacause I live in one of the borders 
+        with the US. When I was little my ideas were influeced by a lot of people 
+        due to my country's environment. When I turned 8 years old, I started to 
+        practice Taekwondo, which was a turning point in my life. That environment
+        showed my different perspectives of life, specially when it came to gender 
+        equality. </p><br>
+
+      <h2> Revelant Projects </h2>
+        <li> [High School] Analyzed lake water from my town's Central's park. The purpose
+            was to research the living beings in the water that were turning the water green
+            as time passed. </li>
+        <li> [UTEP] Worked with linked lists and sorting concepts to translate RNA sequences 
+            to Amino Acid in one of the lab assignments. </li>
+        <li> [UTEP] Worked with stacks and queues by using arithmetic expressions stored in
+            binary trees using postfix notation.</li><br>
+
+      <h2> Contact Information </h2>
+      <a href = "http://linkedin.com/in/diana-valles-57b2571a3"> LinkedIn profile </a><br>
+      <a href = "https://github.com/dianapvalles"> GitHub profile </a><br>
+
+      <div class = "image-thumbnails img">
```

Comment:
you don't need `img`. In your css, when you do `.image-thumbnails img`, it's actually matching on the `<img .../>` tags under the div with `image-thumbnails`, not the `img` string you have here.

Reply:
If you wanted to match an element with both class `image-thumbnails` and `img`, then your css needs to be `.image-thumbnails.img` (no space). Spaces in the css selector means it only matches on child elements.

Reply:
Thank you for the feedback, I'll read more about CSS